client/apps/Admin/src/components/SelectProjectModal.tsx

```
@@ -0,0 +1,117 @@
+            <EditProjectModal
+                projectId={selectedProjectId}
+                onClose={() => {
+                    setSelectedProjectId(null);
+                    onClose();
+                }}
+            />
+        );
+    }
+
+    return (
+        <div className="modal-overlay" onClick={onClose}>
```

Biome (2.1.2)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)